doc/developer/testing.rst

```diff
@@ -9,6 +9,85 @@ Quick instructions
 To run the tests, simply execute the following command. Omit ``SR_PYTHON=On`` or ``SR_FORTRAN=On`` if you do not wish to test those languages:
    ``make test SR_PYTHON=On SR_FORTRAN=On``
 
+####################
+Running single tests
+####################
+
+With the increased automation in the SmartRedis test system, it is now more complex to run
```

Comment:
A reader probably doesn't need to know that their task is harder than it used to be (trees falling in forests). Unless we provide a migration guide, I'd suggest striking the first sentence.

Reply:
Reworked to address this